--- jquery-1.3.2.uncompressed.js	2009-05-06 10:04:18.000000000 +0200
+++ jquery-1.3.2p1.uncompressed.js	2009-05-06 10:02:58.000000000 +0200
@@ -1558,7 +1558,7 @@
 		if ( (match = Expr.match[ type ].exec( expr )) ) {
 			var left = RegExp.leftContext;
 
-			if ( left.substr( left.length - 1 ) !== "\\" ) {
+			if ( typeof(left) != 'undefined' && left.substr( left.length - 1 ) !== "\\" ) {
 				match[1] = (match[1] || "").replace(/\\/g, "");
 				set = Expr.find[ type ]( match, context, isXML );
 				if ( set != null ) {
