--- jquery.form.uncompressed.js.orig	2020-08-12 19:36:51.250355586 +0200
+++ jquery.form.uncompressed.js	2020-08-12 19:37:24.294411983 +0200
@@ -922,10 +922,6 @@
 
 				return (doc && doc.documentElement && doc.documentElement.nodeName !== 'parsererror') ? doc : null;
 			};
-			var parseJSON = $.parseJSON || function(s) {
-				/* jslint evil:true */
-				return window['eval']('(' + s + ')');			// eslint-disable-line dot-notation
-			};
 
 			var httpData = function(xhr, type, s) { // mostly lifted from jq1.4.4
 
@@ -943,7 +939,7 @@
 				}
 				if (typeof data === 'string') {
 					if ((type === 'json' || !type) && ct.indexOf('json') >= 0) {
-						data = parseJSON(data);
+						data = JSON.parse(data);
 					} else if ((type === 'script' || !type) && ct.indexOf('javascript') >= 0) {
 						$.globalEval(data);
 					}
