--- jquery-migrate-1.4.1.js	2016-05-20 03:26:30.000000000 +0200
+++ jquery.migrate.uncompressed.js	2016-05-26 17:19:57.886898431 +0200
@@ -314,9 +314,14 @@
 		/(webkit)[ \/]([\w.]+)/.exec( ua ) ||
 		/(opera)(?:.*version|)[ \/]([\w.]+)/.exec( ua ) ||
 		/(msie) ([\w.]+)/.exec( ua ) ||
+                /(trident).*; rv:([\w.]+)/.exec( ua ) ||
 		ua.indexOf("compatible") < 0 && /(mozilla)(?:.*? rv:([\w.]+)|)/.exec( ua ) ||
 		[];
 
+        if (match[1] === 'trident') {
+          match[1] = 'msie';
+        }
+
 	return {
 		browser: match[ 1 ] || "",
 		version: match[ 2 ] || "0"
