--- jquery.views.uncompressed.js.orig	2020-08-12 19:32:09.349114149 +0200
+++ jquery.views.uncompressed.js	2020-08-12 19:32:45.533266222 +0200
@@ -57,7 +57,7 @@
 }
 
 var $observe, $observable,
-	$isArray = $.isArray,
+        $isArray = Array.isArray || function(obj) { return ({}.toString).call(obj) === "[object Array]"; };
 	$views = $.views;
 
 if (!$.render) {
@@ -79,7 +79,7 @@
 	$sub = $views.sub,
 	$subSettings = $sub.settings,
 	$extend = $sub.extend,
-	$isFunction = $.isFunction,
+        $isFunction = function(ob) { return typeof ob === "function"; };
 	$expando = $.expando,
 	$converters = $views.converters,
 	$tags = $views.tags,
